NAMESPACE

```diff
@@ -122,5 +122,21 @@ export(unv)
 export(v)
 export(weibullDist)
 export(zScores)
+export(rowMean)
```

Comment:
please regenerate this with roxygen2 instead of editing the NAMESPACE file manually.